### DISCUSSION TITLE
Re-implementation of the unpoly-demo in Django

### DISCUSSION THREAD

**Discussion by spapas** [unpoly/unpoly#466] [Show and tell]
Hey friends, I've re-implemented the unpoly demo from https://demo.unpoly.com/ with Django. The repository is at https://github.com/spapas/django-unpoly-demo

All of the original demo functionality has been implemented except the tenants thingiewihch isn't really needed for the unpoly show-off.

The app has also been deployed on render on: https://unpoly-demo.onrender.com/

Enjoy!

**Comment by jmoppel** [comment]
I've played around with this some and it's great. Nice job! Unpoly should host both versions of each demo (along with source code), and give the user a choice if they want to use the Rails or Django version of the app. That would make the demo even more approachable and welcoming.

**Comment by triskweline** [comment]
Awesome work @spapas!

I linked to your Django app on <https://demo.unpoly.com/> (on the start page where it talks about the source code).

**Comment by EugeneDae** [comment]
@spapas Incredible, thank you! I'm just starting out with Unpoly and your code is extremely helpful.

**Comment by spapas** [comment]
Hello friends, because the render.com free tier was way too slow for my liking I deployed the demo on fly.io (unpoly-demo.fly.dev) and added a DNS entry from my domain to it so it won't need to be changed again:

https://unpoly-demo.spapas.net/

**Reply by triskweline** [reply]
Thanks, I updated the link!
